.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -18,6 +18,16 @@ body:
   validations:
     required: true
 
+- type: input
+  attributes:
+    label: Installation method
+    placeholder: |
+      "GitHub releases"
+    description: |
+      How was PowerToys installed? (Examples: Github releases, Microsoft Store, WinGet, Chocolatey, Scoop)
+  validations:
+    required: true
+
```

Comment:
```suggestion
- type: dropdown
  attributes:
    label: Installation method
    description: How / Where was PowerToys installed from?
    multiple: true
    options:
      - GitHub
      - Windows Store
      - PowerToys auto-update
      - Choco
      - Scoop
      - Other
  validations:
    required: true
```

Reply:
does this work?

Reply:
Why `multiple=true`?

Reply:
I guess multiple = true is for the case where they've tried many different installation methods?